keyvault/data-plane/azure-keyvault/src/main/java/com/microsoft/azure/keyvault/requests/ImportCertificateRequest.java

```diff
@@ -132,8 +132,9 @@ private ImportCertificateRequest(Builder builder) {
         password = builder.password;
 
         if (builder.attributes != null) {
-            certificateAttributes = (CertificateAttributes) new CertificateAttributes()
-                    .withNotBefore(builder.attributes.notBefore()).withEnabled(builder.attributes.enabled())
+            certificateAttributes = new CertificateAttributes();
+            certificateAttributes.withNotBefore(builder.attributes.notBefore())
+                    .withEnabled(builder.attributes.enabled())

```

Comment:
I'm not a super big fan of this change. The worry I would have is that we get used to this pattern and run into problems when we encounter a fluent builder pattern (in which case, the second/third lines would have no effect). Is there an option to type it to the parent type instead of the subtype? I'm guessing not, but would be interested to explore if alternate options exist.

Reply:
I am not either. 🤔 I had looked at modifying [Attributes.java](https://github.com/Azure/azure-sdk-for-java/blob/master/keyvault/data-plane/azure-keyvault/src/main/java/com/microsoft/azure/keyvault/models/Attributes.java#L60) somehow to take a `T` (ie. `Attributes<T>`) that but it isn't very retrofit friendly. :/

Would it be better just to check the instance of it... and return that? (blegh. not very beautiful either.)

Reply:
My inclination is to just suppress this and not let it happen in future API design.

Reply:
That works for me. :) I'll revert these.

Reply:
@JonathanGiles How does it look now?